apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2.Snippets/BigQueryClientSnippets.cs

```diff
@@ -528,6 +528,40 @@ public void UploadAvro()
         // See [UploadAvro](ref) for a synchronous example.
         // End see-also
 
+        [Fact]
+        public void UploadParquet()
+        {
+            string projectId = _fixture.ProjectId;
+            string datasetId = _fixture.GameDatasetId;
+            string tableId = _fixture.GenerateTableId();
+
+            var typeInfo = GetType().GetTypeInfo();
+            string resourceName = typeInfo.Namespace + ".us-states.parquet";
+            var stream = typeInfo.Assembly.GetManifestResourceStream(resourceName);
+            // Snippet: UploadParquet(string, string, Stream, *)
+            BigQueryClient client = BigQueryClient.Create(projectId);
```

Comment:
nit: And then here you are using the type because of this being part of the snippet I suppose?

Reply:
Yup.